assets/stylesheets/themes/_dark.scss

```diff
@@ -14,4 +14,29 @@
   h2 {
     color: $headings-color;
   }
+
+  // Tables
+  .table-contrast-bg {
```

Comment:
@zarahzachz These styles should be moved into `assets/stylesheets/components/_tables.scss`.